lib/screens/welcome/splash_screen.dart

```diff
@@ -0,0 +1,48 @@
+import 'dart:async';
+
+import '../../general_exports.dart';
+
+class SplashScreen extends StatefulWidget {
```

Comment:
No need to use stateful , we can use state less and wrap the screen with GetBuilder and create controller for the screen